.ci/s390x/configuration_s390x.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2018 IBM
+#
+# SPDX-License-Identifier: Apache-2.0#
+#
+test:
+  - functional
+  - docker
+
+# Same model as is for aarch64
+docker:
```

Comment:
Comparing these files with the `aarch64` ones, they are *very* similar. I can see that there are small differences but I'm wondering if we can create something like:

```
.ci/generic/*
```

... and have `.ci/aarch64/*` and `.ci/s390x/*` include the generic files if possible. That should work for the shell scripts. I guess it won't work as-is for the YAML as that language doesn't support including natively. But, we could either consider breaking the YAML into chunks and reconstructing or having the scripts use tooling to convert the generic YAML files into arch-specific ones?

Bottom line: it's a maintenance burden on the architecture maintainers to ensure multiple very similar files remain updated. But maybe these files won't change much?

Reply:
hi @jodh-intel now the config looks pretty similar to arm, but on the other hand this file lists (or better black list) what's not working/have been tested on s390x. It could be that in the future the 2 architectures files diverge. On s390x for example, qemu hasn't supported memory hotplug  yet. On s390x, the CPU hotplug has probably a bug with Kata but it should be supported. The file could be a bit redundant but it clearly list what has been tested on s390x. Otherwise, we can have a discussion with @Pennyzct and @nitkon how to integrate the tested features in a more reasonable way for all the arch :)

Reply:
Hi @alicefr - Yes, let's leave these files as separate arch-specific ones for now. We can always re-assess the situation again in the future.